packages/core/src/routes/admin-user.ts

```diff
@@ -264,4 +265,35 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
       return next();
     }
   );
+
+  router.delete(
+    '/users/:userId/identities/:connectorId',
+    koaGuard({ params: object({ userId: string(), connectorId: string() }) }),
+    async (ctx, next) => {
+      const {
+        params: { userId, connectorId },
+      } = ctx.guard;
+
+      const { identities } = await findUserById(userId);
+
+      if (Object.entries(identities).length === 0) {
+        throw new RequestError({ code: 'user.identity_empty', status: 404 });
```

Comment:
Seems unnecessary, empty identities will run into the next error?

By the way, is a common practice to only show necessary messages in error, to prevent data leaking.

Reply:
updated.